src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/NonRazorSdkTests.cs

```diff
@@ -35,7 +35,7 @@ protected override void PrepareProjectForFirstOpen(string projectFileName)
         base.PrepareProjectForFirstOpen(projectFileName);
     }
 
-    [IdeFact(Skip = "https://github.com/dotnet/razor/issues/10965")] // Completion is broken in FUSE for sole `@` characters
+    [IdeFact]
     public async Task Completion_DateTime()
```

Comment:
Integration tests don't run automatically on PRs, right? Should we enqueue a run manually to verify this test?

Reply:
Ran integration tests on this PR and everything looks good.